src/internals/ir/callGraph.ts

```diff
@@ -0,0 +1,169 @@
+import { unreachable } from "../util";
+import { TactASTStore } from "./astStore";
+import { IdxGenerator } from "./indices";
+import { forEachExpression } from "../tact/iterators";
+import {
+  AstFunctionDef,
+  AstReceiver,
+  AstContractInit,
+  AstExpression,
+  AstStaticCall,
+  AstMethodCall,
+} from "@tact-lang/compiler/dist/grammar/ast";
+
+export type CGNodeId = number;
```

Comment:
Let's use the branded type pattern for clarity: https://github.com/nowarp/misti/issues/191